packages/framework/resources/views/layouts/scripts.blade.php

```diff
@@ -21,4 +21,7 @@ function toggleTheme() {
 </script>
 
 {{-- Add any extra scripts to include before the closing <body> tag --}}
-@stack('scripts')
\ No newline at end of file
+@stack('scripts')
+
+{{-- If the user has defined any custom scripts, render them here --}}
+{!! \Hyde\Support\Includes::blade('scripts', config('hyde.hooks.scripts')) !!}
```

Comment:
Optimally, this would support both Blade and Html files, but we don't have an HTML include (yet), and then the coalesce chain would become so long that we'd want to move it to a helper method. So maybe for later.